x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/create_datasource_page/index.tsx

```diff
@@ -85,6 +113,9 @@ export const CreateDatasourcePage: React.FunctionComponent = () => {
     };
     setDatasource(newDatasource);
 
+    if (newDatasource.package && newDatasource.config_id && newDatasource.config_id !== '') {
+      setFormState('VALID');
```

Comment:
Do we have a way to do a better validation?

Reply:
There is full validation in this PR: https://github.com/elastic/kibana/pull/61180

Maybe I should just merge that PR first?

Reply:
Yes :)

Reply:
👍will merge that PR when CI is green

Reply:
The validation PR has been merged. Hope it doesn't cause too many conflicts 😬

Reply:
Just merged master, after fixing a few conflicts ^^